tests/robot-tests/tests/admin_and_public_2/bau/publish_release_and_amend.robot

```diff
@@ -730,11 +730,16 @@ Approve amendment for scheduled release
     set suite variable    ${EXPECTED_PUBLISHED_DATE}
 
 Verify amendment is on Find Statistics page again
+    user waits for caches to expire
     user checks publication is on find statistics page    ${PUBLICATION_NAME}
 
 Navigate to amendment release page
     user clicks link    ${PUBLICATION_NAME}
     user waits until h1 is visible    ${PUBLICATION_NAME}    %{WAIT_MEDIUM}
+
+    user reloads the stale cached page
+    user waits until h1 is visible    ${PUBLICATION_NAME}    %{WAIT_MEDIUM}
```

Comment:
Does kind of suck that we have to reload and sit around (similar thing in other places as well)! I wonder if it'd be possible to fire off a Python request to the page in an earlier part of the test before we arrive at this page?

Probably not necessary for this PR though.

Reply:
I guess hopefully this is just a stop-gap until we have the next version of Next, and then we can invalidate on demand.  At which point, all of these keywords can be mopped up!

Reply:
Yeh good point, hopefully that should be in fairly soon with the frontend deps upgrade.